tests/lean/run/timeParse.lean

```diff
@@ -0,0 +1,203 @@
+import Std.Time
+open Std.Time
+
+def ISO8601UTC : Format .any := date-spec% "YYYY-MM-DD'T'hh:mm:ssZZZ"
```

Comment:
This `Format` definitions also fails on the examples I showed in `date%` above. Then again it is just a test so this is probably fine. I just wanted to point out that this `date-spec` is apparently not conform:
```lean
def foo :=
  let dates := ["2022-07-22T12:52:50.349409+00:00", "2023-12-11T11:31:60.000Z", "2017-06-22T13:32:16Z"]
  dates.map ISO8601UTC.parse
#eval foo -- all evaluate to errors
```

Reply:
It is just an example in the tests just to test some simplified formats. It was not parsing the entire ISO 8601 format, so in the first example you included nanoseconds that were not specified in the format, in the second you have a leap second in a minute and hour that is 23:59:60 that does not configure something valid right now and `ZZZ` does not include `Z`.